lib/_3rdParty/Hunter/config-id/config.cmake

```diff
@@ -0,0 +1,299 @@
+include(hunter_config)
```

Comment:
Also all the files in the `lib/_3rdParty` folder have been created by CMake, please remove them.

Reply:
Same problem as for /lib/CMakeFiles/